pkg/rtc/participant.go

```diff
@@ -589,6 +603,9 @@ func (p *ParticipantImpl) SetMigrateState(s types.MigrateState) {
 		p.pendingOffer = nil
 		// in case of migration, subscriber data channel will not fire OnOpen callback
 		p.activeCounter.CAS(0, 2)
+
+		// use lower network cost to make sure migrated's node has higher priority candidate
+		p.networkCost--
```

Comment:
how would this new network cost transfer to the new node?

Reply:
Also where is this used @cnderrauber . Sorry not able to see it as it is combined with the other large PR.

Reply:
@davidzhao we don't need transfer it to the new node, the higher priority candidate type (peer reflex) will only happen in first connect in our service env(happened when stun server no response or too slow than sfu node). Condition of peer reflex is ice agent got a XOR-MAPPED-ADDRESS that not match to any known address, then a local candidate promote to a peer reflex candidate.

@boks1971 Update now, the code change should be clear.

Reply:
what happens to the second migration? Maybe I'm not understanding the root cause.. but now you'll end up with ICE candidates from both new and old nodes at 99?

Reply:
yeah, they have same cost 99, and new node have new generation, then will fallback to normal case that no problem in other region like sgp, will successed.